src/commands/pack.js

```diff
@@ -187,13 +187,13 @@ function getDevServerConfig() {
   });
 }
 
-function packProject() {
+function packProject(options) {
   return new Promise((resolve, reject) => {
     console.log(
       `${delimiter}: Running webpack...`
     );
     getWebpackConfig().then(
-      (config) => build(config).then(resolve, reject), reject
+      (config) => build(config, options.nobrowser).then(resolve, reject), reject
```

Comment:
I believe there are no problems with passing entire `options` objects through building procedure. But there is a quiestion, if it is necessary to pass positional arguments too? `vorpal` implements them by extending `command` definition like [this](https://github.com/dthree/vorpal/wiki/api-%7C-vorpal.command#action-arguments). So positional arguments are part of `args` structure (near the options). Maybe it is worth to pass full `args` object then, instead of just `args.options`?